alyssa_mckee/Django/amadon/apps/store_front/views.py

```diff
@@ -0,0 +1,28 @@
+from django.shortcuts import render, redirect
+
+# Create your views here.
+def index(request):
+	return render(request, "store_front/index.html")
+def process(req):
+	items_prices = {
+		"shirt":19.99,
+		"sweater":29.99,
+		"duck":4.99,
+		"book":49.99		
+	}
+	if not 'total' in req.session:
+		req.session['total'] = 0
+	if not 'itemcount' in req.session:
+		req.session['itemcount'] = 0
+	
+	price = items_prices[req.POST['item']]
+	
+	req.session['price'] = price*float(req.POST['quantity'])
```

Comment:
Float is decent for money calculations, but at some point you will end up with missing cents. I imagine there is a better data type to use for money. Hint: Decimal

Reply:
right, forgot about that